### ISSUE TITLE
"export 'widget' was not found in '../charting_library.min' when building

### ISSUE THREAD

**Issue by vishnug1995** [tradingview/charting-library-examples#170]
when building electron vuejs it shows "export 'widget' was not found in '../charting_library.min' when building" , but work fine in development

Tradingviewchart.vue
            <template>
                  <div id="tv_chart_container" style="height: 600px;"></div>
              </template>
               <script>
               import { widget } from '../charting_library.min';
                import datafeeds from '../datafeeds/datafees.js'
            export default {
                name: 'tradingviewchart',
                data() {
                    return {
                        widget: null,
                        datafeeds: new datafeeds(this),
                        theme: 'null',
                        interval: 5,
                        openOrders: [],
                        orderLines: [],

                        symbol: 'BNBBTC'
                    }
                },
                mounted() {
                    this.$nextTick(function() {
                        setTimeout(()=>{
                            this.initialize();
                        },1500)
                        

                    });
                },
                methods: {
                    initialize() {
                        console.log(this.widget,"ok");
                        if (!this.widget) {
                           this.widget = new widget({
                                symbol: this.symbol,
                                interval: this.interval,
                                container_id: "tv_chart_container",
                                datafeed: this.datafeeds,
                                library_path: 'static/tradeview/charting_library/',
                                timezone: "Asia/Kolkata",
                                locale: "en",
                                debug: false,
                                autosize: true,
                                custom_css_url: 'chart.css',
                                options:true
                            });

                            this.widget.onChartReady(() => {
                                console.log("ready");
                            });
                       }     

                    },

                }
            }
            </script>

**Comment by timocov** [comment]
Did you looking for similar issues like https://github.com/tradingview/charting-library-examples/issues/80#issuecomment-483326985 or something? `charting_library.min.js` is UMD module, which has export `widget`.

**Comment by TradeRabbit** [comment]
I have been encountering the same error, the only solution so far is to put it in a new, clean project, when it is integrated with an existing project this error happens, I have not yet been able to identify where the incompatibility is;

**Comment by ShadiSimplbit** [comment]
`import { widget } from '../vendor/charting_library/charting_library.esm';`
this works for me

**Comment by CryptoUncleSam** [comment]
Following ShadiSimplbit I did the follow!
Copy charting_library folder into public and src
src --> component --> TVChartContainer open index.jsx
Change ../charting_library.min to the full path of ../charting_library/charting_library.esm.js

**Comment by AKCodez** [comment]
Mine broke as well when updating to V20, i had .../path/to/charting_library/charting_library.cjs and changed it to .../path/to/charting_library/charting_library.esm.js and it began working!

**Comment by timocov** [comment]
usually you don't need to use the full path to a file since charting library has a package.json file where [we have a `main` and `module` fields declared](https://github.com/tradingview/charting_library/blob/ee33bf6476e09cb79779cc417534ed062faff161/charting_library/package.json#L5-L6) so they should be handled by your bundler correctly.

**Comment by AKCodez** [comment]
> 

Yes, but the file has ES-Lint errors, which are so severe that it will not let the bundle compile  without specifying es-lint disable in the file.
